scanner/mappers/mappers.go

```diff
@@ -509,26 +516,26 @@ func fixedInVersion(v *claircore.Vulnerability) string {
 	return fixedIn
 }
 
-// nvdScoresMap look for NVD CVSS in the vulnerability report enrichments and
-// returns a mapping of vulnerability IDs to CVSS Scores V3.
-func nvdScoresMap(enrichments map[string][]json.RawMessage) (map[string]nvdschema.CVSSV30, error) {
-	cvss := enrichments[cvss.Type]
+// nvdVulnerabilities look for NVD CVSS in the vulnerability report enrichments and
+// returns a map of CVEs.
+func nvdVulnerabilities(enrichments map[string][]json.RawMessage) (map[string]nvdschema.CVEAPIJSON20CVEItem, error) {
+	cvss := enrichments[nvd.Type]
 	if len(cvss) == 0 {
 		return nil, nil
 	}
 	// TODO(ROX-21264): ClairCore only supports CVSS v3.1, so we assume CVSSV30 --
 	//                  need to update this when we start using our own CVSS score enricher.
```

Comment:
can be removed (CI probably already flagged this. I haven't checked)

Reply:
851f94a878